README.md

```diff
@@ -44,7 +54,7 @@
 
 5. Open Virtual box, move to appium-test-machine created, select USB and add Android device and Host Controller.
 
-	![alt tag](Appium/virtualbox.png)
+	![alt tag](.github/virtualbox.png)
```

Comment:
Could you move the images to "images" folder? because as far as I know .github folder is for different purpose.

Reply:
Fixed